features/old/api/features.feature

```diff
@@ -8,59 +8,80 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
     When I log in as provider "foo.3scale.localhost"
     And I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
-  # TODO: Test that the "No features yet" notice dissapears when first feature is created,
-  #       and appears when last one is deleted.
+  @javascript
+  Scenario: "No features yet" notice disappears when first feature is created
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    Given the provider has "service_plans" switch allowed
+    And a feature "50% less bugs" of provider "foo.3scale.localhost"
```

Comment:
This step is very confusing as the description is very generic (doesn't say what plan type does the feature belong to - and there are 3 different types), but the implementation is very specific - it assumes not only the plan type - Service - but also searches for service plan name "Basic", and also for some reason assumes that the plan that we are interested is the last one.

```
  @service_plan = provider.service_plans.where(name: "Basic").last
  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
```